src/pages/dev/amplifier/chain-integration/introduction.mdx

```diff
@@ -1,63 +1,101 @@
-    - Build (or instantiate or customize) each of the [required contracts and the relayer](#integration-architecture).
-    - You may choose to change the logic of your verifier contract (for ZK use cases for example) or prover contract.
-1. Whitelist contracts 
-    - For the DevNet [use the form](https://docs.google.com/forms/d/e/1FAIpQLSchD7P1WfdSCQfaZAoqX7DyqJOqYKxXle47yrueTbOgkKQDiQ/viewform) to ask to be whitelisted
-1. Become a [verifier](/validator/amplifier/verifier-onboarding) for your chain.
-    - To test your chain, you will need active verifiers to vote on messages from your chain
-1. Test your integration
-    - Make sure you can send and receive messages on your chain.
-1. Begin audit process
-    - You should audit your on-chain contracts.
-1. Propose to Testnet  
-    - [Make on-chain proposals](/learn/cosmwasm-governance) on the testnet to create your CosmWasm smart contracts.
-        1. Proposal to store your CosmWasm code.
-        1. Proposal to instantiate your contracts.
-    - Make an on-chain proposal to add your chain to the Amplifier router.
-1. Test in Testnet
-    - Ensure your contracts are working as expected.
-    - Verify with developers that they can send messages to and from your chain.
-1. Productionize your relayer
-    - Ensure your relayer is robust, fault-tolerant, and can handle the load of your chain.
-1. Propose to Mainnet
-    - [Make on-chain proposals](/learn/cosmwasm-governance) on Mainnet to create your CosmWasm smart contracts.
-        1. Proposal to store your CosmWasm code.
-        1. Proposal to instantiate your contracts.
-    - Make an on-chain proposal to add your chain to the Amplifier router.
\ No newline at end of file
+## Integration architecture
+
+To connect your chain to the Axelar network via the Interchain Amplifier, you'll need to build or instantiate a [message relayer](/dev/amplifier/chain-integration/introduction#message-relayer) service and the following four smart contracts:
+
+- [Source chain gateway](/dev/amplifier/chain-integration/introduction#source-chain-gateway-contract)
+- [Axelar gateway](/dev/amplifier/chain-integration/introduction#axelar-gateway-contract)
+- [Voting verifier](/dev/amplifier/chain-integration/introduction#verifier-contract)
+- [Multisig prover](/dev/amplifier/chain-integration/introduction#prover-contract)
+
+### Message relayer
+
+- This is a service built with any tech stack that listens for events on your chain and relays them to the Axelar network.
+- For a detailed implementation, see the [Message Relayer example](https://github.com/axelarnetwork/axelar-examples/tree/main/examples/amplifier) on GitHub.
+
+### Source chain gateway contract
+
+- This is the gateway contract for your source chain, built using the same tech stack as that chain. It serves as the primary API for developers sending messages from your chain.
+- The contract must implement the [Cross-Chain Gateway Protocol](https://github.com/axelarnetwork/cgp-spec).
```

Comment:
I believe @milapsheth said this repo might not be up to date anymore, and we should be pointing folks to the Solidity gateway v2.

Reply:
[This one](https://github.com/axelarnetwork/axelar-cgp-solidity/blob/main/contracts/AxelarGateway.sol)?

Reply:
Updated. Should the "EVM chains can re-use the existing EVM Gateway" part go?

Reply:
That's the part that's still correct. Everyone should base off of the EVM gateway here: https://github.com/axelarnetwork/axelar-cgp-solidity/blob/main/contracts/AxelarGateway.sol